app/views/hyrax/homepage/index.html.erb

```diff
@@ -26,7 +26,7 @@
       <i class="fa fa-upload" aria-hidden="true"></i> <%= t('hyrax.share_button') %>
     <% end %>
   <% end %>
-  <p class="terms-of-use"><%= link_to t(:'hyrax.pages.tabs.terms_page'), hyrax.terms_path %></p>
+  <p class="terms-of-use"><%= link_to ('How to Submit'), hyrax.help_path %></p>
```

Comment:
@CB987 You don't need to wrap the text in parentheses here, but it doesn't break anything, so no need to change it.